src/main/java/com/commercetools/sync/commons/utils/QuadriConsumer.java

```diff
@@ -0,0 +1,33 @@
+package com.commercetools.sync.commons.utils;
+
+import java.util.function.Consumer;
+import javax.annotation.Nullable;
+
+/**
+ * Represents an operation that accepts three arguments and returns no result. This is the
+ * three-arity specialization of {@link Consumer}.
+ *
+ * <p>This is a <a href="package-summary.html">functional interface</a>
+ * whose functional method is {@link #accept(Object, Object, Object, Object)}.
+ *
+ * @param <T> the type of the first argument to the function
+ * @param <U> the type of the second argument to the function
+ * @param <S> the type of the third argument to the function
+ * @param <V> the type of the fourth argument to the function
+ * @see Consumer
+ */
+@FunctionalInterface
+public interface QuadriConsumer<T, U, S, V> {
+
+    /**
+     * Performs operation on the given arguments.
+     *
+     * @param firstParam the first argument.
+     * @param secondParam the second argument.
+     * @param thirdParam the third argument.
+     * @param fourthParam the fourth argument.
+     */
```

Comment:
I am not sure if such Java Doc is useful as it does not provide more information than its signature.

Reply:
I just followed pattern used in `TriConsumer` and `BiConsumer`